objectDataTypes.md

```diff
@@ -0,0 +1,149 @@
+
+# Object Data Types
+## Practice Worksheet
+
+
+### Ruby Operations on Basic Object Types
+
+Without putting the code into Ruby or IRB predict the output and data-type of the following?
```

Comment:
Maybe we could give them a table format to make it easier to fill out each answer for each question?